cursor.js

```diff
@@ -127,6 +132,14 @@ WL.registerComponent('cursor', {
     },
 
     update: function() {
+		if(this.doubleClickTimer > 0) {
```

Comment:
We use 4 spaces indentation in code everywhere (sorry! 😉), because it results in more consistent appearance in editors.
It would be great, if you could convert all tabs to 4 spaces.